src/main/configurations/Translate/Configuration_CreeerZaak_LK01.xml

```diff
@@ -276,6 +276,43 @@
                 <Forward name="success" path="EXIT"/>
                 <Forward name="exception" path="EXCEPTION" />
             </SenderPipe>
+            
+            <JwtPipe name="createJwt"
+                expirationTime="600"
+                storeResultInSessionKey="myToken"
+                ifParam="valueAuthType"
+                ifValue="Jwt"
+            >
+                <Param name="valueAuthType" value="${zaakbrug.zgw.zaken-api.auth-type}" />
+                <Param name="sharedSecret" pattern="{password}" authAlias="${zaakbrug.zgw.zaken-api.auth-alias}" hidden="true"/>
+                <Param name="client_id" pattern="{username}" authAlias="${zaakbrug.zgw.zaken-api.auth-alias}" hidden="true"/>
+                <Param name="user_id" pattern="{username}" authAlias="${zaakbrug.zgw.zaken-api.auth-alias}" hidden="true"/>
+                <Param name="user_reresentation" pattern="{username}" authAlias="${zaakbrug.zgw.zaken-api.auth-alias}" hidden="true"/>
+                <Forward name="success" path="DeleteZaakSender" />
+            </JwtPipe>
+
+
+            <SenderPipe name="DeleteZaakSender"
+                getInputFromSessionKey="originalMessage">
+                <HttpSender name="DeleteZaakSender"
+                    methodType="DELETE"
+                    headersParams="Authorization,Accept-Crs,Content-Crs,Accept"
+                    timeout="${zaakbrug.zgw.documenten-api.timeout}"
+                    maxExecuteRetries="5"
+                    contentType="application/json">
+                    <Param name="url" sessionKey="PostZgwZaakResult" xpathExpression="ZgwZaak/url"/>
+                    <Param name="Accept-Crs" value="EPSG:4326" />
+                    <Param name="Accept" value="application/json" />
+                    <Param name="Content-Crs" value="EPSG:4326" />
+                    <!-- <Param name="Host" sessionKey="headers" xpathExpression="/headers/header[@name = 'Host']/text()" /> -->
+                    <!-- <Param name="Host" value="localhost:8080" type="STRING" /> -->
+                    <Param name="Authorization"  sessionKey="originalMessage" xpathExpression="concat('Bearer ', $myToken)">
+                        <Param name="myToken" sessionKey="myToken" defaultValue="@@catalogi-api.jwt@@"/>
+                    </Param>
+                </HttpSender>
+
+                <Forward name="success" path="UncaughtException"/>
+                </SenderPipe>
```

Comment:
Functionally it is correct, but this should be in it's own adapter. You can use any of the other ones as an example(Configuration_Zaken_GetZgwRolTypeByUrl). For delete zaak it would be called `Zaken_DeleteZgwZaak`.